--- Makefile	2011-05-19 09:46:00.000000000 +0900
+++ Makefile	2011-05-19 09:47:12.000000000 +0900
@@ -2,7 +2,7 @@
 
 # BASENAME should point to where the whole lot will be installed
 # change BASENAME to your home directory if need be
-BASENAME	= /usr
+BASENAME	= __PREFIX__
 # For display in the man pages
 VISIBLE_BASENAME= $(BASENAME)
 
@@ -10,7 +10,7 @@
 ARCHITECTURE	=
 #ARCHITECTURE	=.sun4
 
-BINDIR_TAIL	= bin$(ARCHITECTURE)
+BINDIR_TAIL	= bin
-MANDIR		= $(BASENAME)/man
+MANDIR		= $(BASENAME)/share/man
 BINDIR		= $(BASENAME)/$(BINDIR_TAIL)
 VISIBLE_BINDIR	= $(VISIBLE_BASENAME)/$(BINDIR_TAIL)
@@ -112,7 +112,7 @@
 MANS5S	= procmailrc procmailsc procmailex
 
 # Possible locations for the sendmail.cf file
-SENDMAILCFS = /etc/mail/sendmail.cf /etc/sendmail.cf /usr/lib/sendmail.cf
+SENDMAILCFS = /etc/mail/sendmail.cf /etc/sendmail.cf /usr/lib/sendmail.cf __SYSCONFDIR__/sendmail.cf
 
 # Makefile - mark, don't (re)move this, a sed script needs it
 
--- config.h	2011-05-19 09:47:36.000000000 +0900
+++ config.h	2011-05-19 09:50:59.000000000 +0900
@@ -52,7 +52,7 @@
 	is group writable or contained in a group writable home directory
 	if the group involved is the user's default group. */
 
-/*#define LMTP					/* uncomment this if you
+#define LMTP					/* uncomment this if you
 						   want to use procmail
 	as an LMTP (rfc2033) server, presumably for invocation by an MTA.
 	The file examples/local_procmail_lmtp.m4 contains info on how to
@@ -98,7 +98,7 @@
 	mail present.  procmail automatically suppresses this when it isn't
 	needed or under heavy load. */
 
-/*#define DEFsendmail	"/usr/sbin/sendmail"	/* uncomment and/or change if
+#define DEFsendmail	"__SBINDIR__/sendmail"	/* uncomment and/or change if
 						   the autoconfigured default
 	SENDMAIL is not suitable.  This program should quack like a sendmail:
 	it should accept the -oi flag (to tell it to _not_ treat a line
@@ -116,11 +116,11 @@
 	is not found, maildelivery will proceed as normal to the default
 	system mailbox.	 This also must be an absolute path */
 
-#define ETCRC	"/etc/procmailrc"	/* optional global procmailrc startup
+#define ETCRC	"__SYSCONFDIR__/procmailrc" /* optional global procmailrc startup
 					   file (will only be read if procmail
 	is started with no rcfile on the command line). */
 
-#define ETCRCS	"/etc/procmailrcs/"	/* optional trusted path prefix for
+#define ETCRCS	"__SYSCONFDIR__/rcs/"	/* optional trusted path prefix for
 					   rcfiles which will be executed with
 	the uid of the owner of the rcfile (this only happens if procmail is
 	called with the -m option, without variable assignments on the command
@@ -168,7 +168,7 @@
 #define INEFFICIENTrealloc			  /* don't pussy-foot around */
 #endif
 #define MAXinMEM	(1024*1024)		 /* when to switch to mmap() */
-#define MMAP_DIR	"/var/spool/procmail/"		     /* where to put */
+#define MMAP_DIR	"__LOCALSTATEDIR__/spool/procmail/"  /* where to put */
 #endif								/* the files */
 #define MINlogbuf	81			       /* fit an entire line */
 #define MAXlogbuf	1000		       /* in case someone abuses LOG */
