--- sid-filter/Makefile.m4	2006-07-21 06:21:40.000000000 +0900
+++ sid-filter/Makefile.m4	2011-06-08 05:02:39.000000000 +0900
@@ -20,8 +20,8 @@
 
 dnl Enable these next line if needed to specify the locations of libmilter.a
 dnl and the libmilter include files:
-dnl APPENDDEF(`confINCDIRS', `-I/usr/local/sendmail/include')
-dnl APPENDDEF(`confLIBDIRS', `-L/usr/local/sendmail/lib')
+APPENDDEF(`confINCDIRS', `-I/usr/include -I/usr/local/include')
+APPENDDEF(`confLIBDIRS', `-L/usr/lib -L/usr/local/lib')
 
 
 bldPRODUCT_START(`executable', `sid-filter')
--- devtools/M4/UNIX/defines.m4	2006-10-06 14:44:44.000000000 +0900
+++ devtools/M4/UNIX/defines.m4.1	2011-06-10 04:18:08.000000000 +0900
@@ -147,11 +147,11 @@
 
 # Setgid binary ownership/permissions
 GBINOWN=ifdef(`confGBINOWN', `confGBINOWN', `root')
-GBINGRP=ifdef(`confGBINGRP', `confGBINGRP', `smmsp')
+GBINGRP=ifdef(`confGBINGRP', `confGBINGRP', `__SMSUBMITUSER__')
 GBINMODE=ifdef(`confGBINMODE', `confGBINMODE', `2555')
 
 # owner of MSP queue
-MSPQOWN=ifdef(`confMSPQOWN', `confMSPQOWN', `smmsp')
+MSPQOWN=ifdef(`confMSPQOWN', `confMSPQOWN', `__SMSUBMITUSER__')
 
 # MTA binary ownership/permissions
 MBINOWN=ifdef(`confMBINOWN', `confMBINOWN', `root')
--- devtools/OS/GNU	2002-03-22 08:59:25.000000000 +0900
+++ devtools/OS/GNU	2011-06-08 05:41:23.000000000 +0900
@@ -7,7 +7,7 @@
 define(`confOPTIMIZE', `-g -O2')
 define(`confDEPEND_TYPE', `CC-M')
 define(`confEBINDIR', `/libexec')
-define(`confMANROOT', `/man/man')
+define(`confMANROOT', `/share/man/man')
 define(`confMANOWN', `root')
 define(`confMANGRP', `wheel')
 define(`confMANMODE', `644')
--- devtools/OS/Linux	2002-03-22 08:59:25.000000000 +0900
+++ devtools/OS/Linux	2011-06-08 05:42:27.000000000 +0900
@@ -5,9 +5,9 @@
 
 define(`confDEPEND_TYPE', `CC-M')
 define(`confSM_OS_HEADER', `sm_os_linux')
-define(`confMANROOT', `/usr/man/man')
+define(`confMANROOT', `/share/man/man')
 define(`confLIBS', `-ldl')
-define(`confEBINDIR', `/usr/sbin')
+define(`confEBINDIR', `/usr/sbin')
 APPENDDEF(`confLIBSEARCH', `crypt nsl')
 
 define(`confLD', `ld')
